opta/commands/apply.py

```diff
@@ -88,6 +87,7 @@ def apply(
     test: bool,
     auto_approve: bool,
     detailed_plan: bool,
+    var: List[str],
```

Comment:
can you add an example below in `Examples:`?
may be an example with 2 vars to explain the expected syntax